src/ralph/dhcp/models.py

```diff
@@ -0,0 +1,60 @@
+from django.db import models
+from django.utils.timezone import now as time_now
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.networks.models.networks import IPAddress, IPAddressStatus
+
+
+class DHCPEntryManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().select_related('ethernet').exclude(
+            hostname=None,
+            base_object=None,
+            ethernet=None,
+            status=IPAddressStatus.reserved.id
+        )
+
+
+class DHCPEntry(IPAddress):
+    objects = DHCPEntryManager()
+
+    @property
+    def mac(self):
+        return self.ethernet and self.ethernet.mac
+
+    class Meta:
+        proxy = True
+
+
+class DHCPServer(models.Model):
+    ip = models.IPAddressField(verbose_name=_('IP address'), unique=True)
+    last_synchronized = models.DateTimeField(null=True)
+
+    class Meta:
+        verbose_name = _('DHCP Server')
+        verbose_name_plural = _('DHCP Servers')
+
+    @classmethod
+    def update_last_synchronized(cls, ip, time=None):
+        if not time:
+            time = time_now()
```

Comment:
time_now() is misleading, maybe better: timezone.now() ? 

We do not know where it comes from time_now(), I have to look at imports :(